crates/contract/tests/sandbox/sign.rs

```diff
@@ -14,17 +12,22 @@ use mpc_contract::{
 use near_workspaces::types::NearToken;
 use utilities::AccountIdExtV1;
 
-const ECDSA_SIGNATURE_SCHEMES: &[SignatureScheme] =
-    &[SignatureScheme::Secp256k1, SignatureScheme::V2Secp256k1];
+const NON_CKD_SCHEMES: &[SignatureScheme] = &[
+    SignatureScheme::Secp256k1,
+    SignatureScheme::V2Secp256k1,
+    SignatureScheme::Ed25519,
+];
```

Comment:
I hope that at some point we don't need this distinction anymore. Initially I preferred to have CKD separately because the parameters were quite different, and it was not a signature, but they seem to be converging more and more. 

Still, I think we should keep the system open for the introduction of MPC protocols that are not signatures, so maybe it is fine to have at least 2 separate paths for that.